.gitignore

```diff
@@ -3,3 +3,4 @@
 /deps
 erl_crash.dump
 *.ez
+.elixir_ls/
```

Comment:
should have a trailing newline to avoid an extraneous diff later on

Reply:
Fixed